WORKSPACE

```diff
@@ -321,5 +323,84 @@ rbe_autoconfig(
 
 load("@build_bazel_integration_testing//tools:repositories.bzl", "bazel_binaries")
 
-#depend on the Bazel binaries, also accepts an array of versions
-bazel_binaries()
+# Depend on the Bazel binaries
+bazel_binaries(versions = ["0.28.1"])
+
+#
+# Support creating Docker images for our node apps #
+# Not used but needed for @e2e_angular_bazel_example//:bazel_integration_test_files target
+#
+
+http_archive(
+    name = "io_bazel_rules_docker",
+    sha256 = "aed1c249d4ec8f703edddf35cbe9dfaca0b5f5ea6e4cd9e83e99f3b0d1136c3d",
```

Comment:
ugh, it's going to be hard to avoid version skew between this repo and angular-bazel-example if these deps aren't shared via a `package.bzl#devDeps` or something.
Will be fixed by moving angular-bazel-example here, assuming we actually follow through on that